src/test/unit/mcmc/hmc/hamiltonians/ps_point_test.cpp

```diff
@@ -62,10 +63,9 @@ namespace stan {
       stan::test::capture_std_streams();
 
       ps_point point(2);
-      EXPECT_NO_THROW(point.write_metric(0));
-
       std::stringstream out;
-      EXPECT_NO_THROW(point.write_metric(&out));
+      stan::interface_callbacks::writer::stream_writer writer(out);
```

Comment:
Note: this construction is here while we get the writer worked through the more of the code.